cost-analyzer/templates/_helpers.tpl

```diff
@@ -24,6 +24,10 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 {{- end -}}
 
+{{- define "cost-analyzer.serviceName" -}}
+{{- printf "%s-%s" .Release.Name "kubecost-cost-analyzer" | trunc 63 | trimSuffix "-" -}}
```

Comment:
This makes our servicename for the recommended installation path (--name kubecost --namespace kubecost) kubecost-kubecost-cost-analyzer -- it is kind of unintuitive. Can we just do {{ .Release.Name }}-cost-analyzer?